src/streams.c

```diff
@@ -98,6 +98,67 @@ Int READ_COMMAND ( void )
     return 1;
 }
 
+Obj FuncREAD_ALL_COMMANDS( Obj self, Obj stream, Obj echo )
+{
+    ExecStatus status;
+    Int result_count, result_capacity;
+    Obj result, result_list;
```

Comment:
Minor nitpick: Most of the GAP kernel seems to use camelCase, to I find it a bit jarring to see snake_case here.